bindings/imap-codec-python/imap_codec.pyi

```diff
@@ -125,21 +149,42 @@ class GreetingCodec:
         """
 
     @staticmethod
-    def encode(greeting: dict) -> Encoded:
+    def encode(greeting: Greeting) -> Encoded:
         """
         Encode greeting into fragments.
 
-        :param bytes: Given greeting
+        :param greeting: Given greeting
```

Comment:
Is there a way to automatically detect typos for `:param`? (Before it was `dict` vs `bytes`.)

Reply:
I created https://github.com/duesee/imap-codec/issues/572